cypress/e2e/po/side-bars/burger-side-menu.po.ts

```diff
@@ -10,7 +10,7 @@ export default class BurgerMenuPo extends ComponentPo {
    * @returns {Cypress.Chainable}
    */
   static toggle(): Cypress.Chainable {
-    return cy.getId('top-level-menu').should('be.visible').click({ force: true });
+    return cy.getId('top-level-menu').should('exist').click({ force: true });
```

Comment:
I'm a little bit weirded out by jenkins failing because the burger menu isn't visible. It should always be visible... unless there is a loading indicator then it should become visible in the amount of time the `should` waits. This change might mask another issue. To debug is there anyway to view images or videos from the failed tests?

Reply:
I'm no longer experiencing this issue so I can revert this change. If I do see the issue again I'll investigate it further. To answer your question, we don't have a way to view images or videos for Jenkins runs atm; we run the tests locally using docker and enable video/screenshot feature